webrev/src/main/java/org/openjdk/skara/webrev/Webrev.java

```diff
@@ -198,6 +202,11 @@ private void generate(Diff diff, Hash tailEnd, Hash head) throws IOException {
             }
 
             var headHash = head == null ? repository.head() : head;
+            var filesDesc = files.isEmpty() ? "" :
+                            " for files " +
+                            String.join(", ", files.stream().map(Path::toString).collect(Collectors.toList()));
```

Comment:
You could also use [`Collectors.joining`](https://docs.oracle.com/en/java/javase/14/docs/api/java.base/java/util/stream/Collectors.html#joining(java.lang.CharSequence)) here
```suggestion
                            files.stream().map(Path::toString).collect(Collectors.joining(", "));
```